sklearn/utils/__init__.py

```diff
@@ -400,7 +400,7 @@ def safe_sqr(X, copy=True):
     return X
 
 
-def gen_batches(n, batch_size, larger_final_batch=False):
+def gen_batches(n, batch_size, min_batch_size=0):
```

Comment:
The documentation is now out of date

Reply:
added examples and parameter description 👍